joomla/joomla-cms
# [4.0] move icon to block

Pull Request for Issue #29733

### Summary of Changes
move icon to block for webauth button

### Testing Instructions
Access Joomla 4 over a SSL
Go to /administrator/
notice that icon is missing
apply pr.
in termal run `npm ci`
notice that icon is showing in webauth block
notice that the joomla logo is colorized

### Actual result BEFORE applying this Pull Request
![image](https://user-images.githubusercontent.com/1850089/95630130-eae81980-0a46-11eb-93b3-e9961de84d34.png)



### Expected result AFTER applying this Pull Request
![image](https://user-images.githubusercontent.com/1850089/95690889-94164780-0be0-11eb-982f-788bfa223e90.png)




### Documentation Changes Required

none
